.github/workflows/ci.yml

```
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  push:
+    branches: [ main, develop ]
+  pull_request:
+    branches: [ main, develop ]
+
+jobs:
+  빌드:
```

actionlint (1.7.4)

invalid job ID "빌드". job ID must start with a letter or _ and contain only alphanumeric characters, -, or _

(id)